lib/media_player.py

```diff
@@ -28,7 +28,13 @@ def is_playing(self):
 class VLCMediaPlayer(MediaPlayer):
     def __init__(self):
         MediaPlayer.__init__(self)
-        self.instance = vlc.Instance()
+
+        options = (
+          '--video-x=1920 '
+          '--mmal-display hdmi-2'
```

Comment:
Which port is this? The one closest or furthest away from the power plug? Also, can we pass on some mirror mode instead?

Reply:
I'm not sure, 'cause what's printed on the board are HDMI0 and HDMI1.
HDMI0 is the one that's closest to the USB-C power supply port.

For some reason, when I removed the changes from `lib/media_player.py`, video playback is still okay.

Reply:
It seems that video playback will only work if `dtoverlay` is set to `vc4-fkms-v3d` instead of `vc4-kms-v3d` (from `boot/config.txt`).